kroxylicious/src/main/java/io/kroxylicious/proxy/internal/SaslDecodePredicate.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.kroxylicious.proxy.internal;
+
+import org.apache.kafka.common.protocol.ApiKeys;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.kroxylicious.proxy.internal.codec.DecodePredicate;
+
+class SaslDecodePredicate implements DecodePredicate {
```

Comment:
Is this classed used anywhere actually? I can't seem to find any references.

Reply:
Same for the `DecodePredicate` interface itself. Is some part of this missing perhaps from this PR?

Reply:
It will be used in the net filter PR